src/Data/Group/Permutation.hs

```diff
@@ -62,6 +64,7 @@ instance AdditiveGroup a => AdditiveGroup (Permutation a)
 instance AdditiveAbelianGroup a => AdditiveAbelianGroup (Permutation a)
```

Comment:
Above, do you mind fixing the group impl for `Permutation`? I megasmoothbrained it on first pass. It should just be the following: 

```haskell
instance Group a => Group (Permutation a) where	instance Group a => Group (Permutation a) where
  invert (Permutation t f) = Permutation (f . t) (t . f)
``

Reply:
n/m fixed it.